### PR TITLE
Feat/guides link

### PR DESCRIPTION
**Pull request by Steffy29**
Updates Links to xDSL Guides on the Telecom Dashboard page

close #XDSL-5046

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-manager#60.0 ✘  
* Stage 1
  * Build dependencies and inject Sentry ✔ 
* Stage 2
  * Build manager CA ✔ 
  * Build manager EU ✔ 
  * Build manager US ✔ 
  * Tests ✔ 
* Stage 3
  * Prepare tar.gz for UXCI / Release ✘

**Comment by ovh-cds** [comment]
CDS Report build-manager#87.0 ✘  
* Stage 1
  * Build dependencies and inject Sentry ✔ 
* Stage 2
  * Build manager CA ✔ 
  * Build manager EU ✔ 
  * Build manager US ✔ 
  * Tests ✔ 
* Stage 3
  * Prepare tar.gz for UXCI / Release ✘

File: client/app/telecom/dashboard/translations/Messages_fr_FR.xml [hunk 1]
```diff
@@ -1,48 +1,49 @@
...
+  <translation id="telecom_dashboard_services_sms_none" qtlid="305122">Pas de SMS</translation>
+  <translation id="telecom_dashboard_services_hubic_none" qtlid="305135">Pas d'HubiC</translation>
+  <translation id="telecom_dashboard_services_exchange_none" qtlid="305148">Pas d'Exchange</translation>
+  <translation id="telecom_dashboard_services_fax_none" qtlid="305161">Pas de fax</translation>
+
+  <translation id="telecom_dashboard_services_incidents_one" qtlid="305174">incident détecté</translation>
+  <translation id="telecom_dashboard_services_incidents_other" qtlid="305187">incidents détectés</translation>
+
+  <translation id="telecom_dashboard_auth_failed" qtlid="305200">Impossible de récupérer les informations de l'utilisateur.</translation>
+  <translation id="telecom_dashboard_express_lite_desc" qtlid="455712">Express Lite comprend donc un modem offrant un accès à internet et une ligne VoIP pour <span class="ad-price font-weight-bold fs14">22,99€ HT</span>&#160;/&#160;mois.</translation>
```
Comment (FredericEspiau) [inline client/app/telecom/dashboard/translations/Messages_fr_FR.xml]: Wow, hardcoded price ! This should be get from the API !

Comment (FredericEspiau) [reply client/app/telecom/dashboard/translations/Messages_fr_FR.xml]: I know, I'm just surprised, we will look upon it 😉

**Comment by Steffy29** [comment]
I need to remove this branch and to restart from /develop branch, too many changes are shown from the rebase action. :-/

**Pull request closed without merging**